src/main/java/seedu/storemando/logic/parser/SortCommandParser.java

```diff
@@ -21,9 +22,13 @@ public SortCommand parse(String args) throws ParseException {
         String trimmedArgs = args.trim();
         if (trimmedArgs.equalsIgnoreCase(EXPIRYDATE_KEYWORD)) {
             return new SortExpiryDateCommand();
-        } else if (trimmedArgs.equalsIgnoreCase(QUANTITY_ASCENDING_KEYWORD)) {
+        }
+        String[] splitWords = trimmedArgs.split("\\s+");
+        if (splitWords.length == 2 && splitWords[0].equalsIgnoreCase(QUANTITY_KEYWORD)
+            && splitWords[1].equalsIgnoreCase(ASCENDING_KEYWORD)) {
             return new SortQuantityCommand(true);
```

Comment:
I think SortQuantityCommand shouldn't be taking in a boolean :)
1 suggestion I can think of is to create 2 separate classes handling asc and desc. I think there is a more elegant way though :)

Reply:
alright will do so!